### PR TITLE
Feature: deletetg command

### LINKED ISSUE #54: Delete a Tutorial Group from Student
As a CS TA, I can delete a class from a student so that I can make sure a student has the correct classes

### PR DESCRIPTION
**Pull request by jaysmyname**
resolves #54

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/DeleteTutorialGroupCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_invalidStudentIndex_failure() {
+        // index out of bounds
+        showStudentAtIndex(model, INDEX_FIRST_STUDENT);
+        Index outOfBoundIndex = INDEX_SECOND_STUDENT;
+        // ensures that outOfBoundIndex is still in bounds of address book lit
+        assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getStudentList().size());
```
Comment (LowJiaHao99) [inline src/test/java/seedu/address/logic/commands/DeleteTutorialGroupCommandTest.java:37]: How can outOfBoundIndex still be within bounds of address book?

Perhaps a better name for the testcase could be execute_InvalidStudentIndexWithFilteredListOfOneStudent_failure()

perhaps an elaboration of why it is out of bound while it is within addressbook can be written
(possible elaboration as comment for Index outOfBoundIndex: //index is larger than size of filteredlist which is 1

File: src/test/java/seedu/address/logic/parser/DeleteTutorialGroupParserTest.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        // zero
+        assertParseFailure(parser, "0" + TUTORIAL_GROUP_DESC_CS2103T_W15_3, MESSAGE_INVALID_FORMAT);
+    }
+
+    @Test
+    public void parse_invalidValue_failure() {
+        // empty tutorial group
+        assertParseFailure(parser, "1" + TUTORIAL_GROUP_EMPTY, TutorialGroup.MESSAGE_CONSTRAINTS);
+
+        // multiple invalid values, but only the first invalid value is capture
```
Comment (LowJiaHao99) [inline src/test/java/seedu/address/logic/parser/DeleteTutorialGroupParserTest.java:55]: Should it be //invalid value ?

**Review by LowJiaHao99 — CHANGES_REQUESTED** [review]
Some changes required for testcase elaborations, otherwise LGTM

File: src/test/java/seedu/address/logic/parser/DeleteTutorialGroupParserTest.java (hunk 2 above)
Comment (jaysmyname) [reply src/test/java/seedu/address/logic/parser/DeleteTutorialGroupParserTest.java:55]: empty tutorial group is still an invalid value, also line 56 has invalid index and invalid tutorial group

**Review by LowJiaHao99 — APPROVED** [review]

**Comment by LowJiaHao99** [comment]
LGTM

**Pull request merged**